queries/liquidsoap/highlights.scm

```diff
@@ -0,0 +1,69 @@
+(op) @operator
+"..." @operator
+["and" "or" "not"] @keyword.operator
+
+[ "def"
+  "fun"
+  "let" ] @keyword.function 
```

Comment:
Keep styling consistent, have brackets on their own line and use 2-space indentation for items

Reply:
Done!